custom/icds_reports/models/util.py

```diff
@@ -14,6 +14,15 @@ class AggregateSQLProfile(models.Model):
     name = models.TextField()
     date = models.DateField(auto_now=True)
     duration = models.PositiveIntegerField()
+    latest_aggregation = models.DateField(null=True)
+
+    @classmethod
+    def get_last_indicator_acknowledged(cls):
```

Comment:
it is getting the last one only. I think the function is little misguiding. Moreover this function is not required for this change. Since fetching from this model will only required when FE will ask for it. Therefore removing this function since it is never called.